src/modules/core/components/toast/toast.css

```diff
@@ -0,0 +1,51 @@
+:root {
+  --toastify-color-success: rgba(104, 210, 210, 1) !important;
```

Comment:
Can we pull these colours from `src/styles/variables.css` in all cases.

Reply:
👍